charts/opentelemetry-collector/values.yaml

```diff
@@ -80,6 +80,8 @@ presets:
 configMap:
   # Specifies whether a configMap should be created (true by default)
   create: true
+  # Specifies whether the configMap should be mounted by chart name (false by default) 
```

Comment:
Can you update this comment to more explicitly list out what this setting does

Reply:
Sure do you think the name should be updated as well something like `mountOverride`? @TylerHelmuth

Reply:
Let's name it `includeMount`

Reply:
curious, why isn't creating a configMap a sufficient condition for it to be mounted in the deployment/daemonset/statefulset?

Reply:
Mainly for the use case when a user is bringing their own config